docker-entrypoint.sh

```diff
@@ -20,4 +20,8 @@ if [[ -d "/opt/blueprints/nginx" ]]; then
     echo "Volume mount exists copying files"
     cp -r shelf-layouts/ nginx/
     cp -r external-frames/ nginx/
-fi
\ No newline at end of file
+fi
```

Comment:
Can we remove the volume mount check and copy?
I don't think that it is something that we use?

Reply:
You mean the shelf-layouts part of it, right? Because the external frame... where would nginx take it from otherwise?

Reply:
Valid point. I forgot about the external frame. In that case it is fine to keep it.